tools/samtools.wdl

```diff
@@ -637,3 +652,61 @@ task collate_to_fastq {
         maxRetries: max_retries
     }
 }
+
+task fixmate {
+    meta {
+        description: "This WDL task runs Samtools fixmate on the input BAM file. This fills in mate coordinates and insert size fields."
+    }
+
+    parameter_meta {
+        bam: "Input BAM format file to add mate information. Must be name-sorted or name-collated."
+        prefix: "Prefix for the output file. The extension will be added."
```

Comment:
```suggestion
        prefix: "Prefix for the output file. The extension `.bam` will be added."
```

Reply:
Suggested this before seeing the next line. What I suggested is wrong; we need to a bit more verbose here